notebooks/text_models/solutions/text_generation_using_transformers.ipynb

```diff
@@ -0,0 +1,911 @@
+{
```

Comment:
<ul><li>"scaled down transformer decoder based generative text model" --&gt; "scaled-down transformer-decoder-based generative text model" (maybe too many qualifying words before "model" here)</li><li>Explain what "causal masking" is</li><li>I'd remove the sentence "The layer has no cross-attention when run with decoder sequence only." which reads a bit confusing without more explanation.</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/372/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
<p>Thank you! Updated! </p>